dbus_service.py

```diff
@@ -403,11 +403,10 @@ def check_and_enrich_ahoy_data(self, meter_data):
          # Check for Attribute (inverter)
         if (self._servicename == "com.victronenergy.inverter" and
                 not "fld_names" in meter_data):
-            raise ValueError("Response from OpenDTU does not contain fld_names in data")
+            raise ValueError("Response from ahoy does not contain fld_names in data")
         # Check for an additonal Attribute
         if not "ch0_fld_names" in meter_data:
-            raise ValueError("Response from OpenDTU does not contain ch0_fld_names data")
-        # not needed: meter_data["record"] = self.fetch_ahoy_record_data()
+            raise ValueError("Response from ahoy does not contain ch0_fld_names data")
```

Comment:
Ich würde gerne die "not needed" Zeile drin lassen, bitte, weil das evtl. mal wieder relevant wird und für vollständige Daten sorgen würde. Vielleicht ändern in "Record data is currently not needed here..."

Reply:
habe ich wieder dazu gemacht

Reply:
Wenn du noch Zeit und Nerv hast, dann setze noch "Record data is currently not needed here" ein.